pyanaconda/payload/manager.py

```diff
@@ -112,7 +112,7 @@ def add_listener(self, event_id, func):
             elif event_id <= self._thread_state:
                 func()
 
-    def restart_thread(self, payload, fallback=False, checkmount=True, onlyOnChange=False):
+    def restart_thread(self, payload, fallback=False, checkmount=True, on_change=False):
```

Comment:
I think the "only" is needed to make sense. Alternatively, you could flip the logic and call it something like  `skip_additions`.

Reply:
I am not sure what skipping additions means. The thread will be restarted only if it looks like the repositories have changed... `verify_change`? `force_restart`?

Reply:
Point taken. Anything against keeping the "only" part?

Also - if this blocks something for you, feel free to go ahead, the explanation in doctext is good enough, even if the name does not help me personally.

Reply:
That seems to be the best option. Changed.